rocketmq-spring-boot/src/main/java/org/apache/rocketmq/spring/metric/MetricExtension.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.rocketmq.spring.metric;
+
+public interface MetricExtension {
+
+    /**
+     * Add current count of message from the producer.
+     *
+     * @param topic topic name
+     * @param count count of message
+     */
+    void addProducerMessageCount(String topic, int count);
```

Comment:
int or long should be consistent.

Reply:
The parameter type is always int, and long is not used.